plone/app/locales/locales/it/LC_MESSAGES/plone.po

```diff
@@ -217,7 +217,7 @@ msgstr "Eventi nei prossimi 7 giorni."
 
 #: CMFPlone/controlpanel/browser/relations_inspect.pt:40
 msgid "<strong>Warning!</strong> You have broken relations! You can <a href=\"${portal_url}/@@rebuild-relations\">inspect and rebuild all relations</a>."
-msgstr ""
+msgstr "<strong>Attenzione!</strong> Ci sono delle relazioni interrotte! Puoi <a href=\"${portal_url}/@@rebuild-relations\">vedere e ricostruire tutte le relazioni</a>."
```

Comment:
"relazioni interrotte" non e' bellissimo ma non mi viene in mente nulla di meglio :)
CC @pnicolli @cekk avete qualche suggerimento?